bundles/config/org.eclipse.smarthome.config.dispatch/src/main/java/org/eclipse/smarthome/config/dispatch/internal/ConfigDispatcher.java

```diff
@@ -226,15 +221,17 @@ public int compare(File left, File right) {
                 }
             });
             for (File file : files) {
-                try {
-                    processConfigFile(file);
-                } catch (IOException e) {
-                    logger.warn("Could not process config file '{}': {}", file.getName(), e.getMessage());
-                }
+                processConfigFile(file);

```

Comment:
This will now recursively process all subdirectories from `services`. Is this intended? You may just call `internalProcessConfigFile` here to restore the original behaviour.

Reply:
Good catch! I had it like that before and thought that this can be done "easier", but I was wrong. Thanks for pointing me to that mistake!